README.md

```diff
@@ -1,3 +1,3 @@
-# My first project
+# My first project is great
```

Comment:
I think this change is important.

Reply:
I'm going to disagree with that unfortunately